rdt/hyper_transformer.py

```diff
@@ -21,7 +21,14 @@ def __repr__(self):
             'sdtypes': self['sdtypes'],
             'transformers': {k: repr(v) for k, v in self['transformers'].items()}
         }
-        return json.dumps(config, indent=4)
+
+        printed = json.dumps(config, indent=4)
+        for transformer in self['transformers'].values():
+            quoted_transformer = f'"{transformer}"'
```

Comment:
Is there any chance that the quotes would be single quotes? If so should we check for that?

Reply:
I think not since the accepted standart by `JSON` is `"`.